apps/emqx_conf/src/emqx_conf_schema.erl

```diff
@@ -1151,6 +1151,7 @@ translation("emqx") ->
 translation("gen_rpc") ->
     [
         {"default_client_driver", fun tr_gen_rpc_default_client_driver/1},
+        {"ssl_client_port", fun tr_gen_rpc_ssl_client_port/1},
```

Comment:
this is probably the only change needed.

or fix this instead: gen_rpc has a default value for ssl clients to connect to 5370
https://github.com/emqx/gen_rpc/blob/46dd22100391f3d87daca989cde7e2ecc52a17b5/src/gen_rpc.app.src#L35

but we have set the same server port on 5369 for both tcp and ssl:
https://github.com/emqx/emqx/blob/a4ab744d4ed47526f889718163830eeb9c77359e/apps/emqx_conf/src/emqx_conf_schema.erl#L798-L813

Reply:
for completeness, we should actually add `tcp_client_port` too

Reply:
actually, to support `manual` strategy properly, it should not be a simple translation and user server port. 
should rather make the tcp and ssl ports configurable for the clients.

Reply:
if we force all nodes in the cluster to use the same server port, then maybe just a translation is enough.